js/app.js

```diff
@@ -0,0 +1,193 @@
+  { "title" : "You Only Live Twice", "year" : 1967, "actor" : "Sean Connery", "gross" : "$756,544,419" },
+  { "title" : "The Spy Who Loved Me", "year" : 1977, "actor" : "Roger Moore", "gross" : "$692,713,752" },
+  { "title" : "Casino Royale", "year" : 2006, "actor" : "Daniel Craig", "gross" : "$669,789,482" },
+  { "title" : "Moonraker", "year" : 1979, "actor" : "Roger Moore", "gross" : "$655,872,400" },
+  { "title" : "Diamonds Are Forever", "year" : 1971, "actor" : "Sean Connery", "gross" : "$648,514,469" },
+  { "title" : "Quantum of Solace", "year" : 2008, "actor" : "Daniel Craig", "gross" : "$622,246,378" },
+  { "title" : "From Russia with Love", "year" : 1963, "actor" : "Sean Connery", "gross" : "$576,277,964" },
+  { "title" : "Die Another Day", "year" : 2002, "actor" : "Pierce Brosnan", "gross" : "$543,639,638" },
+  { "title" : "Goldeneye", "year" : 1995, "actor" : "Pierce Brosnan", "gross" : "$529,548,711" },
+  { "title" : "On Her Majesty's Secret Service", "year" : 1969, "actor" : "George Lazenby", "gross" : "$505,899,782" },
+  { "title" : "The World is Not Enough", "year" : 1999, "actor" : "Pierce Brosnan", "gross" : "$491,617,153" },
+  { "title" : "For Your Eyes Only", "year" : 1981, "actor" : "Roger Moore", "gross" : "$486,468,881" },
+  { "title" : "Tomorrow Never Dies", "year" : 1997, "actor" : "Pierce Brosnan", "gross" : "$478,946,402" },
+  { "title" : "The Man with the Golden Gun", "year" : 1974, "actor" : "Roger Moore", "gross" : "$448,249,281" },
+  { "title" : "Dr. No", "year" : 1962, "actor" : "Sean Connery", "gross" : "$440,759,072" },
+  { "title" : "Octopussy", "year" : 1983, "actor" : "Roger Moore", "gross" : "$426,244,352" },
+  { "title" : "The Living Daylights", "year" : 1987, "actor" : "Timothy Dalton", "gross" : "$381,088,866" },
+  { "title" : "A View to a Kill", "year" : 1985, "actor" : "Roger Moore", "gross" : "$321,172,633" },
+  { "title" : "License to Kill", "year" : 1989, "actor" : "Timothy Dalton", "gross" : "$285,157,191" }
+];
+
+let bondTitles = [];
+
+// console.log(bondFilms.length);
+
+let onlyTitles = () => {
+	for (let i = 0; i < bondFilms.length; i++) {
+		bondTitles.push(bondFilms[i].title)
+	}
+}
+
+onlyTitles();
+
+console.log(bondTitles);
+
+let oddBonds = [];
+
+let onlyOdds = () => {
+	for (let i = 0; i < bondFilms.length; i++) {
+		if (bondFilms[i].year % 2 === 1) {
+			oddBonds.push(bondFilms[i])
+		}
+	}
+}
+
+onlyOdds();
+
+console.log(oddBonds);
+
+
+
+let grossArray = [];
+
+let totalGross = 0 ;
+
+let calculateGross = () => {
+	for (let i = 0; i < bondFilms.length; i++) {
+		grossArray.push(bondFilms[i].gross);
+	}	
+	// for (let j = 0; j < grossArray.length; j++) {
+	// 	grossArray[j].replace(/$/g, " ");
+	// 	grossArray[j].replace(/,/g, " ");
+	// 	parseInt(grossArray[j], 10);
+	// }
+}
+
+let toNum = () => {
+	for (let j = 0; j < grossArray.length; j++) {
+		grossArray[j].replace(/$/g, " ");
```

Comment:
if this were working you wouldn't know it because .replace doesn't mutate the data-- you have to assign its output to something (or assign it back to grossArray[j] in your case